### ISSUE TITLE
`keybase version` gives error if server isn’t started

### ISSUE THREAD

**Issue by lucaswerkmeister** [keybase/client#1923]
```
$ keybase ctl stop
$ keybase version
▶ ERROR dial unix /run/user/1000/keybase/keybased.sock: connect: no such file or directory
```
Most other commands seem to start the server if it isn’t running already. Alternatively, it might make sense for this command to just print the client version if the server isn’t running. But this error shouldn’t happen, I think.

I’m on version 1.0.10-0 (Arch package).

**Comment by maxtaco** [comment]
Thanks! This was by design but another bug crept in that prevents it from
displaying the service isn't running before crapping out.

**Comment by maxtaco** [comment]
Should be fixed in our latest update (1.0.10). Thanks for the bug report.

**Comment by lucaswerkmeister** [comment]
Can confirm it’s fixed in 1.0.11. Thanks!
